openid-connect-4-credential-issuance-1_0.md

```diff
@@ -382,9 +457,84 @@ Location: https://client.example.net/cb?
     &state=af0ifjsldkj
 ```
 
+## Dynamic Credential Request
+
+This step is OPTIONAL. This section describes how the Issuer can obtain additional credentials from the End-user after receiving an Authorization Request. The Issuer MUST utilize [@OIDC4VP] and [@SIOPv2] to dynamically request additional credentials.
+
+This provides the benefit of the Issuer being able to adhere to the principle of data minimization, for example by including only minimum requirements in the Credentiam Manifest knowing that it can supplement additional information if needed.
+
+### Additional Parameters in the Authorization Request
+
+In order to enable dynamic callbacks of the issuer to the end-user's wallet, the wallet MAY provide the following additional parameters in the authorization request: 
+
+* `wallet_issuer`: OPTIONAL. JSON String containing the wallet's OpenID Connect Issuer URL. The issuer will use the discovery process as defined in [@SIOPv2] to determine the wallet's capabilities and endpoints. 
+* `user_hint`: OPTIONAL. JSON String containing an opaque usr hint the wallet MAY use in sub-sequent callbacks to optimize the user's experience. 
+
+Note: credential issuer's client_id with wallet
+
+For a non-normative example of Request and Response, see section 11.6 in [@!OIDC4VP].
+
+## Pushed Authorization Endpoint
+
+This step is OPTIONAL. Authorization requests in this specification follow the definition given in Section 3.1.2 of [@!OpenID], with the exception of the differences specified in the Simple Issuance Flow of this document. In case of complex flows or when the size of the request is large due to the query language or encrypting, clients are RECOMMENDED to use pushed authorization requests.
+
+### Pushed Authorization Request
+
+Authorization Requests are made as defined in Section 3.1.2.1 of [@!OpenID]. This specification defines the following requirements beyond [@!OpenID]:
+
+The authorization request MUST include the `claims` parameter defined in section 5.5 of [@!OpenID] with a new top-level element `credentials`. 
+
+* `credentials`: REQUIRED. JSON array where every object describes a credential being requested and (optionally) may contain references to verifiable 
```

Comment:
restructured so that the we currently have examples of standard authz request and PAR as part of the authorization endpoint section. example of other grant types (e.g. CIBA) can be added later.